### PR TITLE
10182 add module paymet_wcpay

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by elicoidal** [comment]
This one should be proposed to OCA?

**Comment by Rona111** [comment]
not the latest version.

File: payment_wcpay/__init__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
```
Comment (siyuancn) [inline payment_wcpay/__init__.py:4]: wrong copyright information

File: payment_wcpay/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2015 Elico Corp (<http://www.elico-corp.com>)
+#    Chen Rong <chen.rong@elico-corp.com>
+#
```
Comment (siyuancn) [inline payment_wcpay/__openerp__.py:7]: wrong copyright information

File: payment_wcpay/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Wechat Payment Acquirer',
+    'category': 'Hidden',
+    'summary': 'Payment Acquirer: Wechat Implementation',
+    'version': '1.0',
```
Comment (siyuancn) [inline payment_wcpay/__openerp__.py:26]: wrong module version

**Comment by siyuancn** [comment]
@Rona111 Please update the copyright information.

**Comment by noahzaozao** [comment]
@elicoidal @seb-elico For wcpay, since the PR OCA/l10n-china are not able to merge , https://github.com/OCA/l10n-china/pull/11 , shall we merge this PR? I'll create new PR with my changes after this one.

**Comment by elicoidal** [comment]
I would update the wc and alipay in OCA/l10n-china and try to push to review.
I will send a mail to the community to allow us to merge the modules between us as well (kind of optimistic merging within OCA)

**Comment by elicoidal** [comment]
Let's focus on the OCA/l10n-china

**Pull request closed without merging**